rules.rst

```diff
@@ -0,0 +1,43 @@
+LUG Rules
+=========
+
+Preamble
+--------
+This is a set of rules for the Oregon State University Linux Users Group. These
+are not set in stone and are in fact meant to be dynamic; adapting to the needs
+of the community as they change. These rules are unique from our constitution
+(which is also in this repository) by still enforced by the community.
+
+For information on contributing to the set of rules see the `Contributing`
+section below.
+
+Rules
+-----
+- There should be a maximum of one bot in the irc channel (#osu-lug,
+  irc.freenode.net) at any time. 
+
+Contributing
+------------
+How to propose a rule:
+0. Have a GitHub account.
```

Comment:
for reference, http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#enumerated-lists

```
1. thing one
2. thing two
```